.travis.yml

```diff
@@ -124,4 +124,4 @@ script:
       PYTHONPATH=$PWD:$PYTHONPATH py.test tests/test_documentation.py;
     else
       PYTHONPATH=$PWD:$PYTHONPATH py.test tests/ --ignore=tests/integration_tests --ignore=tests/test_documentation.py --ignore=tests/keras/legacy/layers_test.py --cov-config .coveragerc --cov=keras tests/;
-    fi
+    fi
```

Comment:
keep the same with dev branch

Reply:
It is the same as dev - for some reason its showing that there is a diff.

Reply:
Diff is an extra line at end of file is deleted.